pages/kids/stories/index.tsx

```diff
@@ -57,28 +71,38 @@ const StoriesSection = ({ stories }: StoriesSectionProps) => {
 
         <Link
           href={`/kids/stories/${story.slug}`}
-          className="w-2/5 bg-primary-blue rounded-2xl text-white text-[10px] sm:text-[12px] lg:text-base p-2 w-3/5 text-center mr-4">
-
+          className="w-2/5 bg-primary-blue rounded-2xl text-white text-[10px] sm:text-[12px] lg:text-base p-2 w-3/5 text-center mr-4"
+        >
           {t('kids_page.playStory')}
-
         </Link>
       </div>
     </div>
   );
 
+  const renderStoriesSection = (language: string, stories: Story[], titleKey: string) => {
+    // Filter stories that have a title in the current language
+    const localizedStories = stories.filter((story) => story.title[language]);
+
+    return (
+      <>
+        <h2 className="text-primary-blue text-center">{t(titleKey, { defaultValue: 'stories' })}</h2>
+        {localizedStories.map((story) => renderStoryPanel(story))}
+      </>
+    );
+  };
+
   return (
     <div className="bg-gradient-to-r from-[#fdf6d2] to-[#99bde4] -mb-8 -m-2">
       <SEO
         title={t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
         description={t(`seo.kids_page_storiesDescription.${getCountryVariant()}`)}
         image="https://www.movapp.cz/icons/movapp-cover-kids.jpg"
       />
-      {getCountryVariant() === 'cs' ? (
+      {countryVariant === 'cs' || countryVariant === 'sk' ? (
         <div className="min-h-screen m-auto py-10 px-2 sm:px-4">
-          <h2 className="text-primary-blue text-center">{t('kids_page.czechStories')}</h2>
-          {stories_CZ.map((story) => storyPanel(story))}
-          <h2 className="text-primary-blue text-center mt-12">{t('kids_page.ukrainianStories')}</h2>
-          {stories_UK.map((story) => storyPanel(story))}
+          {storiesByCountry.CZ?.length > 0 && renderStoriesSection('cs', storiesByCountry.CZ, 'kids_page.czechStories')}
```

Comment:
Do we need to check for all countries here? Isn't it enough to render just `uk` stories and `getCountryVariant()` stories?

Reply:
I think this is separation "Ceske lidove rozpravky", "Ukrajinske lidove rozpravky" and based on discussion with Martin, we will have also "origin unknown" :)

Reply:
fixed, see comment above